app/models.py

```diff
@@ -1001,6 +1003,16 @@ def __init__(self, **kwargs):
     postage = db.Column(db.String, nullable=True)
     broadcast_data = db.Column(JSONB(none_as_null=True), nullable=True)
 
+    welsh_content = db.Column(db.Text)
+    welsh_subject = db.Column(db.Text)
+    languages = db.Column(
```

Comment:
question: do you think we should be more specific in naming these, eg `welsh_letter_content`, to make it readily apparent that these only apply to letters? 🤔 

optional suggestion:
```suggestion
    welsh_letter_content = db.Column(db.Text)
    welsh_letter_subject = db.Column(db.Text)
    letter_languages = db.Column(
```

Reply:
Ooh good idea, I will change this throughout.